src/apiCalls.js

```diff
@@ -37,10 +37,23 @@ const fetchRecipes = () => {
         });
 }
 
+const fetchIngredients = () => {
+  fetch(`https://what-s-cookin-starter-kit.herokuapp.com/api/v1/ingredients`)
+  .then(response => response.json())
+  .then(ingredientData => {pageData.allIngredients = ingredientData.ingredients})
+  .catch(error => console.log(error))
```

Comment:
oh and shouldn't it be `console.error(error)`?

Reply:
Gosh what would I do without you?